src/app.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I assume you used it for `next: NextFunction` at line 23. You can delete this disable comment and change this part as `_next: NextFunction`. But do not forget to add the rule below rule to your eslint config file.
`    "@typescript-eslint/no-unused-vars": [
      "error",
      { "argsIgnorePattern": "^_" }
    ]
`